scripts/gen_large_room.py

```
@@ -356,8 +361,8 @@ def add_event(event_type, state_key, sender, content, prev=None):
 # Phase 6: Final Churn + Wrap Up (events ~8000-10000)
 # ============================================================================
 print("Phase 6: Final churn...")
-remaining = NUM_EVENTS - len(events)
-for i in range(remaining):
+REMAINING = NUM_EVENTS - len(events)
+for i in range(REMAINING):
```

Ruff (0.16.2)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)